ci/build/build-pipeline.yaml

```diff
@@ -256,31 +256,43 @@ spec:
       plan:
       - get: cloudhsm-config
         trigger: true
-      - task: build-cloudhsm
-        privileged: true
-        config:
-          <<: *build_config
-          params:
-            CONTEXT: cloudhsm-config/cloudhsm
-          inputs:
-          - name: cloudhsm-config
-      - put: cloudhsm-client-image
-        params:
-          <<: *image_put_params
-          additional_tags: cloudhsm-config/.git/short_ref
-
-      - task: build-cloudhsm-jce
-        privileged: true
-        config:
-          <<: *build_config
-          params:
-            CONTEXT: cloudhsm-config/cloudhsm/jdk-jce-image
-          inputs:
-          - name: cloudhsm-config
-      - put: cloudhsm-jce-image
-        params:
-          <<: *image_put_params
-          additional_tags: cloudhsm-config/.git/short_ref
+      - in_parallel:
+          limit: 3
+          fail_fast: true
+          steps:
+          - do:
+            - task: build-cloudhsm
+              privileged: true
+              config:
+                <<: *build_config
```

Comment:
I think use of this anchor is bit confusing at this point... maybe the anchor should just be for the `image_resource` part (since everything else params/inputs/outputs appears to be overwritten in each use of the task) so it's a bit misleading and looks like some magic merging is going on